mindjunkies/courses/models.py

```
@@ -72,19 +73,28 @@ class Course(BaseModel):
-    verified = models.BooleanField(default=False)
-
     def __str__(self):
         return self.title
 
     def save(self, *args, **kwargs):
         if not self.slug:
-            self.slug = slugify(self.title)
+            base_slug = slugify(self.title)
+            slug = base_slug
+            # Check if slug already exists
+            ModelClass = self.__class__
```

Ruff (0.8.2)

Variable `ModelClass` in function should be lowercase

(N806)